### PR TITLE
Prepare content-loader tool for community view

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Prepare content-loader tool for community view:
  - make more generic content-loader tool for community repo purpose

**Related epic**
https://github.com/kyma-project/website/issues/356

**Related issue**
https://github.com/kyma-project/website/issues/392

### CODE REVIEW

**Comment by aerfio** [comment]
Maybe we could also error if APP_ZEN_HUB_TOKEN and APP_TOKEN are not present.
And we have two config files now, one in src and one in prepare-roadmap - each API key is declared in different config 😞

File: tools/content-loader/src/prepare-community/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  if (err) {
+    throw new VError(
+      err,
+      `while cloning ${coreConfig.organization}/${coreConfig.repository}`,
+    );
+  }
+
+  [err] = await to(CopyCommunity.do(tempPath, outputPath));
+  if (err) {
+    throw err;
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply tools/content-loader/src/prepare-community/index.ts]: `err` from `CopyCommunity.do` function contains right message, so writing new message is unnecessary for me.

**Comment by magicmatatjahu** [comment]
@aerfio We have also separate config for docs content :) I want keep hermetic config to each content: docs, community, roadmap. APP_ZEN_HUB_TOKEN is declared in roadmap config because only there is needed, but APP_TOKEN is needed for git client, and content-loader uses it in every domain :)

**Comment by aerfio** [comment]
OK! One last thing - we have two prettier configs, it's unnecesary

File: tools/content-loader/README.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ The configuration options are as follows:
 | **APP_DOCS_OUTPUT**                     | **YES**  | `docs`                   | The path for storing the documentation results                        |
 | **APP_DOCS_VERSIONS_CONFIG_FILE**       | **YES**  | `versions.json`          | The path to the website documentation configuration                   |
 | **APP_DOCS_TEMP_DIR**                   | **YES**  | `tempDocsDir`            | The path for storing temporary data for documentation                 |
+| **APP_COMMUNITY_OUTPUT**                | **YES**  | `community`              | The path for storing the community content results                    |
+| **APP_COMMUNITY_REPOSITORY**            | **YES**  | `community`              | The repository with community                                         |
```
Comment (mmitoraj) [inline tools/content-loader/README.md]:
```suggestion
| **APP_COMMUNITY_REPOSITORY**            | **YES**  | `community`              | The repository with the community content                                         |
```

File: tools/content-loader/README.md [hunk 3]
```diff
@@ -36,6 +36,9 @@ The configuration options are as follows:
...
+| **APP_COMMUNITY_TEMP_DIR**              | **YES**  | `tempCommunityDir`       | The path for storing temporary data for community content             |
```
Comment (mmitoraj) [inline tools/content-loader/README.md]:
```suggestion
| **APP_COMMUNITY_TEMP_DIR**              | **YES**  | `tempCommunityDir`       | The path for storing temporary data for the community content             |
```

**Pull request merged**